### PR TITLE
Fixes #16000 - Show correct state in listing product content for an activation key.

### PR DESCRIPTION
**Pull request by ranjan**

### CODE REVIEW

**Comment by ranjan** [comment]
It depends on katello/katello#6232

File: lib/hammer_cli_katello/activation_key.rb [hunk 1]
```diff
@@ -40,7 +40,8 @@ def request_params
         params = super
         if options.keys.any? { |o| o.match(/\Aoption_organization.*/) }
           params['organization_id'] = resolver.organization_id(
-            resolver.scoped_options('organization', all_options))
+            resolver.scoped_options('organization', all_options)
+          )
```
Comment (akofink) [inline lib/hammer_cli_katello/activation_key.rb]: This seems like an unnecessary change.

**Comment by akofink** [comment]
The change looks good, but please add a test for the product-content sub-command.

File: lib/hammer_cli_katello/activation_key.rb (hunk 1 above)
Comment (ranjan) [inline lib/hammer_cli_katello/activation_key.rb]: The above indentation change is suggested by rubocop.

**Comment by akofink** [comment]
LGTM - ACK Thanks @ranjan!

**Comment by ranjan** [comment]
Thanks @akofink, Just for information Is it ok to merge this commit without waiting for the PR Katello/katello#6232 to be merged?.

**Comment by akofink** [comment]
@ranjan Good catch; That's my bad. The tests will still pass on hammer-cli-katello because they run against a cached API, but it will break master branch for hammer until Katello/katello#6232 is merged. As for downstream, the two changes will be cherry-picked in together because they're on the same issue.

For the future, it helps to put a [WIP] in the title of the hammer PR until the dependencies are met.
